### DISCUSSION TITLE
Converting GitHub profile to Organisation does not copy across the Description or Links.

### DISCUSSION THREAD

**Discussion by BeverlyCode** [community/community#83757] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When you add a GitHub profile to your existing account and then convert it to an Organisation the description is not copied across from the Account to the Organisation, neither are the rest of the profile details such as links. This would be easy to fix and I don't see a valid reason why it would not do this.

**Comment by SohamJack003** [comment]
Response #1
